wagtailio/areweheadlessyet/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+from wagtail.admin.edit_handlers import FieldPanel, MultiFieldPanel
+from wagtail.core.fields import RichTextField
+from wagtail.core.models import Page
+
+
+class AreWeHeadlessYetHomePage(Page):
```

Comment:
Should we reuse this project’s `SocialMediaMixin`? And potentially other mixins?

Reply:
I've added it along with the `CrossPageMixin`.